api/resolvers/search.js

```diff
@@ -194,7 +194,16 @@ export default {
 
       const { query, quotes, nym, url } = queryParts(q)
 
-      if (url) {
+      if (url && !query.length) {
+        const domain = url.startsWith('url:www.') ? url.slice(8) : url.slice(4)
+        const value = `*${domain}*`
+        const fields = ['title', 'text']
+
+        const queries = fields
+          .map((field) => ({ wildcard: { [field]: { value, case_insensitive: true } } }))
+
+        whatArr.push({ bool: { should: queries } })
+      } else if (url) {
```

Comment:
I don't think we need the wildcards on title and text. The analyzer we use on title and text you can perform a normal `match`-like query on a url, e.g. https://stacker.news/search?q=geyser.fund, so maybe we can just add the domain to the `query` variable after it's parsed?

Also more generally, [wildcards are slow](https://opensearch.org/docs/latest/query-dsl/term/wildcard/) and best to avoid them on large fields like `text` if we can.

Reply:
I've tried adding the domain back to the query variable after parsing but to no effect, I'm curious if the tokenizer used in production is separate from the default one used for development?

Am I correct that the index is built here:  [`curl -X PUT "http://localhost:9200/item" -ku admin:admin`](https://github.com/stackernews/stacker.news/blob/65cc1dbcc0294f513f4da708889bf907593077f9/docker-compose.yml#L94)

Still investing a more performant fix.

Reply:
Ah yes, locally you'll get a default mapping and `standard` analyzer for the index if you don't set a mapping with an `english` analyzer first. The semantic search doc has an example mapping, which if you remove the ml stuff is:

```json
{
  "mappings": {
    "properties": {
      "text": {
        "type": "text",
        "analyzer": "english",
        "fields": {
          "keyword": {
            "type": "keyword",
            "ignore_above": 256
          }
        }
      },
      "title": {
        "type": "text",
        "analyzer": "english",
        "fields": {
          "keyword": {
            "type": "keyword",
            "ignore_above": 256
          }
        }
      }
    }
  }
}
```

Reply:
Awesome, this helped a lot.